tests/lib/snaps/account-control-consumer/bin/useradd

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+exec /usr/sbin/useradd "$@"
```

Comment:
also ... inside the classic chroot the password database is fully writable so you should not need to access extrausers for development work inside the chroot it is not actually clear which type of "classic" @teknoraver refers to above ...